packages/documentation/gatsby-config.js

```diff
@@ -10,6 +10,26 @@ module.exports = {
   },
   pathPrefix: '/veteran-facing-services-tools',
   plugins: [
+    {
+      resolve: `@gatsby-contrib/gatsby-plugin-elasticlunr-search`,
+      options: {
+        // Fields to index
+        fields: [`title`, `tags`],
+        // How to resolve each field`s value for a supported node type
+        resolvers: {
+          // For any node of type MarkdownRemark, list how to resolve the fields` values
+          MarkdownRemark: {
+            title: node => {
+              console.log(node);
```

Comment:
I think everything is wired up correctly but the search is returning no results. I don't really understand the gatsby pipeline but AFAIK none of the nodes that this gets are the mdx content. It's mostly just comments from jsx files. 

![Screen Shot 2019-04-25 at 14 37 22](https://user-images.githubusercontent.com/4998130/56766862-a8098780-6767-11e9-9ea6-b210fa42c342.png)

I tried to delete my cache and find content from the mdx files I'd added front matter to but no luck. 

My understanding is that this needs access to the content nodes so it can build an index from the front matter fields.